alerta-covid/db/migrate/20210329014814_create_symptoms.rb

```diff
@@ -0,0 +1,9 @@
+class CreateSymptoms < ActiveRecord::Migration[6.1]
+  def change
+    create_table :symptoms do |t|
+      t.string :name
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
Hmmmm not sure if is really needed to store this in the DB.

It can leave in a Symptom model without an actual db, but store them in `ymls` or something like that
It's just my opinion. 

Do we have any plans for managing this from an admin interface or something?

Reply:
we plan to use the symptom table to relate it to the incidents table, can we do it using ymls?

We do not plan to manage symptoms